### PR TITLE
fix #443 bug

### LINKED ISSUE #443: [Eclipse+IntelliJ] Using the C5 basic and C5 standard pricing tier to create the redis cache failed
[Eclipse+IntelliJ2017+IntelliJ2016]

Repro steps:
Create a redis cache and Pricing tier choose C5 Basic or C5 Standard
Click OK , then a no content error dialog occured.
image

**(A Redis cache with the above configuration can be created successfully on the portal)**
@jdneo

### PR DESCRIPTION
**Pull request by jdneo**
1. remove useless lines in FormUtil.java
2. fix #443 bug

### CODE REVIEW

**Comment by jdneo** [comment]
@adashen @Eskibear @testforstephen @bsaby

File: Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/rediscacheprocessors/RedisCacheCreator.java [hunk 1]
```diff
@@ -59,8 +59,8 @@ private void initCreatorsForPremiumTier(RedisCaches redisCaches, String dnsName,
     	}
     }
     public RedisCacheCreator(RedisCaches redisCaches, String dnsName, String regionName, String groupName) {
-    	initCreatorsForBasicTier(redisCaches, dnsName, regionName, groupName, new int[] {0, 1, 2, 3, 4, 6});
-    	initCreatorsForStdTier(redisCaches, dnsName, regionName, groupName, new int[] {0, 1, 2, 3, 4, 6});
+    	initCreatorsForBasicTier(redisCaches, dnsName, regionName, groupName, new int[] {0, 1, 2, 3, 4, 5, 6});
+    	initCreatorsForStdTier(redisCaches, dnsName, regionName, groupName, new int[] {0, 1, 2, 3, 4, 5, 6});
```
Comment (adashen) [inline Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/rediscacheprocessors/RedisCacheCreator.java]: what does this int array mean?

Comment (jdneo) [inline Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/rediscacheprocessors/RedisCacheCreator.java]: This means the capacity of the redis cache service.
i.e.  Basic has C0 - C6 capacities, so the array is ranging from 0 to 6

Comment (adashen) [inline Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/rediscacheprocessors/RedisCacheCreator.java]: enum will be better?

Comment (jdneo) [inline Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/rediscacheprocessors/RedisCacheCreator.java]: Actually as we have discussed before, I'm going to change the redis cache creation logic in next sprint. So I guess this code snippet will disappear in future

Comment (adashen) [inline Utils/azuretools-core/src/com/microsoft/azuretools/azurecommons/rediscacheprocessors/RedisCacheCreator.java]: ok

**Pull request merged**